### DISCUSSION TITLE
Mobile app design

### DISCUSSION THREAD

**Discussion by StephanieLongden** [UKHomeOffice/design-system#457] [Ideas]
We should have a standardised design system or guidelines for designing mobile apps. The Home Office has a few public facing apps (EU Exit, ID check and the future ETA app).

Would be good to make sure we are consistent across gov with how we are designing for mobile apps, so that we don't inadvertently start creating different versions of the design system. 

**What already exists**
Examples of mobile app design systems across gov:
HMRC mobile design system [ https://hmrc.invisionapp.com/console/share/EXOVJCPYDPN/328697735](url)

Below are some examples of how we currently use native Android and IOS components on our apps, as well as how we are proposing to use the gov.uk design system components on the future ETA app. 

**ID Check app:**
![ID check app](https://user-images.githubusercontent.com/92519123/191503093-b0faa269-b1bf-42b4-9733-a5dedc2c3856.png)

**EU Exit app:**
![EU exit](https://user-images.githubusercontent.com/92519123/191500430-a42ab460-d278-4a78-97cc-01b232df51d1.png)

**ETA app design [Not live]:** 
![ETA](https://user-images.githubusercontent.com/92519123/191500444-3130ede8-f37d-4a19-8364-e7acbcb39f29.png)

Please contribute any examples of gov apps to this discussion.

**Comment by tomyems** [comment]
This probably fits alongside the idea of our patterns pages, where there is discussion and guidance about what to do.

# Mobile application
A sentence or two about what is a mobile app
## When to use a mobile application

## When not to use a mobile application

## Accessibility?

## Research?

## Components
Explain about native components

### Radios
Explain about native android radios, and then link to the HMRC radios for iOS?

### Text input

and so on?

What do you think? Feel free to improve, suggest or rewrite? It might be too much for one page, not sure at this point?
